### ISSUE TITLE
Code reproducibility

### ISSUE THREAD

**Issue by dgjung0220** [JianghaiSCU/Diffusion-Low-Light#5]
I trained your code with LOLv1, but I couldn't achieve performance like in the experiments. Did you also use the synthetic data of 1,000 images mentioned in the paper, in addition to LOLv1's 485 images?

**Comment by JianghaiSCU** [comment]
yes, we adopted the full training data of LOLv1 to optimize our network

**Comment by dgjung0220** [comment]
The results of training with LOLv1 485 images are as follows:
![image](https://github.com/JianghaiSCU/Diffusion-Low-Light/assets/16290330/385efbfd-6e65-46a9-a5f2-d4d2ce43b5b4)

The pretrained model I am providing did not show a grid. I am using the training code and settings available on GitHub. Is there something I might have missed? Below are the results of the pretrained model. It indeed doesn't seem to show a grid pattern.

![image](https://github.com/JianghaiSCU/Diffusion-Low-Light/assets/16290330/f1f05085-8945-4ec8-b69d-35fa13d86fdb)

**Comment by JianghaiSCU** [comment]
It seems that the model does not converge, maybe you can try to extend the training iterations.

**Comment by dgjung0220** [comment]
Your config code specifies 500 iterations, and I followed it as is, but as you mentioned, it seems that convergence hasn't been achieved yet, as you suggested. Thank you for your response. I'll try increasing the number of iterations to see if that helps.

**Comment by lbl0342** [comment]
Hello, I also found the same problem, may I ask whether the effect of adjusting the epoch has improved

**Comment by zhenyuan1234** [comment]
Hi, where can I download the extra 1000 images, thanks!

**Comment by lbl0342** [comment]
Are you asking for synthetic data from LOLv1？




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年12月5日(星期二) 上午9:44
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [JianghaiSCU/Diffusion-Low-Light] Code reproducibility (Issue #5)





 
Hi, where can I download the extra 1000 images, thanks!

**Comment by lbl0342** [comment]
Are you asking for synthetic data from LOLv1？
BMVC2018 Deep Retinex 分解 (daooshee.github.io)


------------------ 原始邮件 ------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年12月5日(星期二) 上午9:44
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [JianghaiSCU/Diffusion-Low-Light] Code reproducibility (Issue #5)





 
Hi, where can I download the extra 1000 images, thanks!

**Comment by zhenyuan1234** [comment]
Yes, I have found the 1000 synthetic data, tkanks! However, the recovered image still has the grid even though it has been iterated for 200000 steps.  Is there something I am missing to cause this undesired result.

**Comment by JianghaiSCU** [comment]
> Yes, I have found the 1000 synthetic data, tkanks! However, the recovered image still has the grid even though it has been iterated for 200000 steps. Is there something I am missing to cause this undesired result.

Hi, you could try removing the TV loss during early training and adding it for the last few epochs.

**Comment by zhenyuan1234** [comment]
Hi, why are the visual results tested with your pre-trained model different in your paper.

**Comment by JianghaiSCU** [comment]
> Hi, why are the visual results tested with your pre-trained model different in your paper.

The visual results were not all generated using the same pre-trained model, and the released model is for reference only.
